session.go

```diff
@@ -88,6 +88,9 @@ type Session struct {
 	tabletsRoutingV1 bool
 
 	usingTimeoutClause string
+
+	tracingInfoFetchAttempts int
+	tracingInfoFetchInterval time.Duration
```

Comment:
Let's move it to the tracer, and use retry policy interface instead.

Reply:
What do you mean by using retry policy interface? I do not see how `RetryPolicy` methods would be useful in this case 
```
Attempt(RetryableQuery) bool
GetRetryType(error) RetryType
```

Reply:
My bad I was thinking about `ReconnectionPolicy` but let's don't create confusion and keep your solution as it is, only move these variables to the tracer.